src/aws-documentation-mcp-server/pyproject.toml

```
@@ -111,6 +111,7 @@ update_changelog_on_bump = true
 packages = ["awslabs"]
 
 [tool.pytest.ini_options]
+python_files = ["test_*.py", "integ.*.py"]
```

Is the `integ.*.py` necessary?
```suggestion
python_files = ["test_*.py"]
```

---

No actually I'll remove it since the pattern changed. Not using the dot naming anymore. Python can't import it because dots in module filenames are interpreted as package separators. Will remove the whole line.